packages/jest-resolve/src/index.js

```diff
@@ -187,9 +186,25 @@ class Resolver {
       } catch (ignoredError) {}
     }
 
-    // 4. Throw an error if the module could not be found. `resolve.sync`
-    //    only produces an error based on the dirname but we have the actual
-    //    current module name available.
+    return null;
+  }
+
+  resolveModule(
+    from: Path,
+    moduleName: string,
+    options?: ResolveModuleConfig,
+  ): Path {
+    const dirname = path.dirname(from);
+    const module = this._resolveModuleFromDirIfExists(
+      dirname,
+      moduleName,
+      options,
+    );
+    if (module) return module;
+
+    // Throw an error if the module could not be found. `resolve.sync`
```

Comment:
Can we add the number back? This is a part of a group of commits, would be nice not to lose the context

Reply:
Not sure if that makes it more or less confusing - maybe `(4.)` to indicate that it is an "optional" 4th step, executed in the `resolveModule` code path but not if `_resolveModuleFromDirIfExists` is called directly?